core/unit_test/TestRangePolicyConstructors.hpp

```diff
@@ -74,13 +74,38 @@ TEST(TEST_CATEGORY_DEATH, range_policy_invalid_bounds) {
   using Policy    = Kokkos::RangePolicy<TEST_EXECSPACE>;
   using ChunkSize = Kokkos::ChunkSize;
 
-  ASSERT_DEATH({ (void)Policy(100, 90); },
-               "Kokkos::RangePolicy bounds error: The lower bound \\(100\\) is "
-               "greater than the upper bound \\(90\\)\\.");
+  char msg[] =
+      R"WARNING(Kokkos::RangePolicy bounds error: The lower bound (100) is greater than the upper bound (90).
+)WARNING";
```

Comment:
> R"WARNING(Kokkos::RangePolicy

Is `R` at front a typo or is it for a regex magic?

Reply:
It is a [raw string literal](https://en.cppreference.com/w/cpp/language/string_literal).
I went for it when I saw all the character escaping but now I realize none of it is actually needed.